### DISCUSSION TITLE
Profile Status irreversible invisibility issue with ":" symbol

### DISCUSSION THREAD

**Discussion by yunfachi** [community/community#101476] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When inserting the symbol ":" into the status, it becomes invisible to you, even after changing it, but remains visible to others. Reproduction steps:
1. Insert the symbol ":" into the status (for example, the status "hello `:trollface:`" is suitable, but you can also use just ":").
2. Change the status to any other, for example, "hello" (without using ":").

Now you won't see your status, for example, "hello" but if you log in from incognito or another account, you will see it. From another account or incognito, you can also see the status with `:`.

It's worth noting that this is a frontend bug. Notably, in the sidebar menu, the status is clearly visible; it becomes invisible only when hovering over the emoji in the profile.

<details>
<summary>...</summary>

Cleared the cache and logged in again. Nothing changed. The issue persists on two different accounts. I checked it on Firefox 121.0b9 (64-bit) Developer Edition for NixOS. This issue is not observed in the mobile version (android), as expected.
</details>

**Comment by ghost** [comment]
I came here to report something similar:

When assigning text to my profile status, the text doesn't appear on hover while I'm logged in. It will appear if I'm logged out.

This is the relevant portion of HTML when I'm logged out:
![github_logged_out](https://github.com/community/community/assets/156826133/86cfa634-ced3-493e-9e33-9993ac1719da)

And this is the HTML generated when I'm logged in:
![image](https://github.com/community/community/assets/156826133/d98fbd5d-3725-405a-9b80-246fd5f7828e)

I'm assuming that it has something to do with the button rendering, because other accounts can visit my account and check my status message. It's only when I visit my page when I'm logged it that the hover effect doesn't work.
